app.py

```diff
@@ -10,23 +10,50 @@
 #from flask import Flask, render_template, request, jsonify, Blueprint
 
 import web
+from web import form
 import signal
 from json import dumps
+import pymongo
```

Comment:
Do you really need to import the whole module? Can you try to use `from ... import ...` instead for specific objects to import?